src/components/OutdoorWeatherWidget.js

```diff
@@ -0,0 +1,54 @@
+import { useEffect, useState } from "react";
+import { ReactComponent as Sun } from "../icons/sun.svg";
+import { ReactComponent as Cloudy } from "../icons/sun.svg";
+import { ReactComponent as PartlyCloudy } from "../icons/cloud-sun.svg";
+import { ReactComponent as Rain } from "../icons/cloud-rain.svg";
+import { ReactComponent as Thunder } from "../icons/cloud-lightning.svg";
+
+export default function OutdoorWeatherWidget() {
+  const [weather, setWeather] = useState();
```

Comment:
you could have state to handle if the widget is loading and if the widget has errored, and then render it accordingly.

https://codesandbox.io/s/usefetch-6boe7?file=/src/App.js

in that codesandbox you can see what I mean